src/adapter.ts

```diff
@@ -271,31 +271,18 @@ export class MongooseAdapter {
    * @param {Object} model Casbin model to which policy rule must be loaded
    */
   loadPolicyLine(line: policyLine, model: Model) {
-    let lineText = line.p_type;
-
-    if (line.v0) {
-      lineText += ', ' + line.v0;
-    }
-
-    if (line.v1) {
-      lineText += ', ' + line.v1;
-    }
-
-    if (line.v2) {
-      lineText += ', ' + line.v2;
-    }
-
-    if (line.v3) {
-      lineText += ', ' + line.v3;
-    }
-
-    if (line.v4) {
-      lineText += ', ' + line.v4;
+    let lineArray: Array<string> = [];
+    lineArray.push(line.p_type!);
+
+    for (const word of [line.v0, line.v1, line.v2, line.v3, line.v4, line.v5]) {
+      if (word !== undefined) {
+        lineArray.push(word)
+      } else {
+        break
```

Comment:
If we follow the same logic as [here](https://github.com/node-casbin/sequelize-adapter/blob/master/src/adapter.ts#L63), it seems to me that function is not allowed to break if the `word` is `undefined`. Not sure if this is correct, but maybe @hsluoyz can elaborate.

And you are already iterating over an array, creating a new array which you then join seems unnecessary to me ;)

I would rather keep the normal string addition.

Reply:
I think https://github.com/node-casbin/sequelize-adapter/blob/master/src/adapter.ts#L63 has the same bug with https://github.com/node-casbin/mongoose-adapter/issues/46, I will later fix it.